Fabric.Terminology.TestsBase/Fake/AsyncEnumerator.cs

```diff
@@ -4,6 +4,10 @@
     using System.Threading;
     using System.Threading.Tasks;
 
+#pragma warning disable CA1063 // Implement IDisposable Correctly
```

Comment:
`IEnumerator<T>` implements `IDisposable`. Are you sure we should be ignoring this?

Reply:
@j2jensen I did find an issue already reported in NullGaurd.Fody - https://github.com/Fody/NullGuard/issues/90.  I am intending to add the dependency again once the issue is resolved.  You might notice that I did not remove Fody from any of the supporting libraries (as of yet).  Reasoning here, I was able to step through the test projects and the API to finish the upgrade to dotnet core/standard 2.0 without needing to dig too deep.    Also, the current effort to wire in Fabric.Identity and Fabric.Authorization will mostly be done in the API project (or maybe a new project if we find there is enough that we can share with other teams).  

I will go ahead and clean up the [AllowNull] attributes now that Fody has been removed as it will be good to work through the issues explicitly when it is reintroduced.

As for fulling implementing IDisposable on the Fake (test) - I felt is was overkill in that situation since most of the code would be essentially default anyway.